tests/src/Behat/WysiwygContext.php

```diff
@@ -52,15 +52,16 @@ public function assertNoWysiwyg($label) {
    *
    * @Then I should see the wysiwyg button :button in the field :field
    */
-  public function iShouldSeeWysiwygButton($field, $button) {
+  public function iShouldSeeWysiwygButton(string $field, string $button): void {
     $wysiwyg = $this->getWysiwyg($field);
-    $button_elements = $this->getSession()->getDriver()->find($wysiwyg->getXpath() . '//a[@title="' . $button . '"]');
-    if (empty($button_elements)) {
-      throw new \Exception("Could not find the '$button' button.");
-    }
-    if (count($button_elements) > 1) {
-      throw new \Exception("Multiple '$button' buttons found in the editor.");
```

Comment:
Why did we lose this assertion?

Reply:
You can't have the same button multiple times in an editor. Config doesn't allow it.